src/main/java/uk/ac/ebi/atlas/experimentpage/ExpressionAtlasContentService.java

```diff
@@ -0,0 +1,83 @@
+            singleCellBaselineExperimentExternallyAvailableContentServiceGeo;
+    private final ExternallyAvailableContentService<SingleCellBaselineExperiment>
+            singleCellBaselineExperimentExternallyAvailableContentServiceEna;
+    private final ExternallyAvailableContentService<SingleCellBaselineExperiment>
+            singleCellBaselineExperimentExternallyAvailableContentServiceEga;
+    private final ExperimentTrader experimentTrader;
+
+    public ExpressionAtlasContentService(
+            LinkToArrayExpress.SingleCell singleCellLinkToArrayExpress
+            ,
+            LinkToEna.SingleCell singleCellLinkToEna,
+            LinkToEga.SingleCell singleCellLinkToEga,
+            LinkToGeo.SingleCell singleCellLinkToGeo,
+            ExperimentTrader experimentTrader) {
+        this.experimentTrader = experimentTrader;
+
+        this.singleCellBaselineExperimentExternallyAvailableContentService=
+                new ExternallyAvailableContentService<>(ImmutableList.of(singleCellLinkToArrayExpress));
+
+        this.singleCellBaselineExperimentExternallyAvailableContentServiceGeo =
+                new ExternallyAvailableContentService<>(ImmutableList.of(singleCellLinkToGeo));
+
+        this.singleCellBaselineExperimentExternallyAvailableContentServiceEna =
+                new ExternallyAvailableContentService<>(ImmutableList.of(singleCellLinkToEna));
+
+        this.singleCellBaselineExperimentExternallyAvailableContentServiceEga =
+                new ExternallyAvailableContentService<>(ImmutableList.of(singleCellLinkToEga));
+    }
+
+    public List<ExternallyAvailableContent> list(String experimentAccession,
+                                                 String accessKey,
+                                                 ExternallyAvailableContent.ContentType contentType) {
+        Experiment<?> experiment = experimentTrader.getExperiment(experimentAccession, accessKey);
+        String externalResourceType = externalResourceLinksPriority(experiment);
+
+        List list = singleCellBaselineExperimentExternallyAvailableContentService.list(
+                (SingleCellBaselineExperiment) experiment, contentType);
+        List list2 =  externalResourceType.equals("geo") ?
+                singleCellBaselineExperimentExternallyAvailableContentServiceGeo.list(
+                (SingleCellBaselineExperiment) experiment, contentType) :
+                externalResourceType.equals("ega") ?
+                        singleCellBaselineExperimentExternallyAvailableContentServiceEga.list(
+                        (SingleCellBaselineExperiment) experiment, contentType) :
+                        singleCellBaselineExperimentExternallyAvailableContentServiceEna.list(
+                        (SingleCellBaselineExperiment) experiment, contentType);
+        list2.addAll(list);
+        return list2;
+    }
+
+    private String externalResourceLinksPriority(Experiment<?> experiment) {
```

Comment:
this method is returning `externalResourceType` string so can we change this method's name to something like `getExternalResourceType(...)`?